app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -68,6 +70,16 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
      */
     private var uiLifecycleCancelJob: Job
 
+    /**
+     * Setting the openContextMenu function in the HomeTileAdapter is fragile
+     * since we need to use Delegates.observable to update the function. However,
```

Comment:
Why do we need to use Delegates.observable? Add to the comment.

nit: the comment's a little confusing too: "the function will be updated by then". When's then? Consider rewording it.